src/display/editor/editor.js

```diff
@@ -1059,8 +1067,11 @@ class AnnotationEditor {
     }
     this._editToolbar = new EditorToolbar(this);
     this.div.append(this._editToolbar.render());
-    if (this.#altText) {
-      await this._editToolbar.addAltText(this.#altText);
+    const toolbarButtons = this.toolbarTools;
+    if (toolbarButtons) {
+      for (const [name, tool] of toolbarButtons.reverse()) {
```

Comment:
Why is the array being processed in reverse order here, as it would seem simpler to always provide the toolbar buttons in order in the array data structure?